vlmeval/vlm/emu.py

```diff
@@ -28,6 +28,13 @@ def __init__(self,
         from transformers import AutoModelForCausalLM, AutoTokenizer
         from accelerate import init_empty_weights, infer_auto_device_map, dispatch_model
         
+        local_rank,world_size = get_rank_and_world_size()
+        
+        device_1 = local_rank
+        device_2 = local_rank+world_size
+        torch.cuda.set_device(device_1)
+        torch.cuda.set_device(device_2)
```

Comment:
Add some check / assertion to make sure such arrangement is viable. 
For example, check if you have 2 * world_size GPUs